pkg/trace/api/api_test.go

```diff
@@ -954,6 +954,23 @@ func BenchmarkWatchdog(b *testing.B) {
 	}
 }
 
+func TestReceiverTimeoutConfig(t *testing.T) {
+	tcs := [][]int{{1, 2, 1, 2}, {1, 0, 1, 15}, {0, 1, 5, 1}, {0, 0, 5, 15}}
+	for _, tc := range tcs {
+		t.Run("receiver timeout", func(t *testing.T) {
```

Comment:
```suggestion
		t.Run("", func(t *testing.T) {
```
In Go it's usually just one word for the identifier. In this case, if you leave it blank, Go will just add numbers for each test, like an index, which is good enough here.